__tests__/app.test.js

```diff
@@ -140,4 +140,40 @@ describe('GET Requests', () => {
             });
         });
     });
+});
+
+describe('DELETE Requests', () => {
+    describe('DELETE /api/comments/:comment_id', () => {
+        test('/api/comments/:comment_id 200: responds with deleted comment when ID is valid', () => {
+            return request(app)
+            .delete('/api/comments/1')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.comment).toEqual(expect.objectContaining({

```

Comment:
This is great if you want to send back the deleted comment to the user - you could alternatively just send back a 204 status which doesn't (no content) but does have a more descriptive status code. Depends on what you want your user experience to be